pytgcalls/pytgcalls/mtproto/pyrogram_bridge.py

```diff
@@ -105,21 +105,21 @@ async def check_group_call(self) -> bool:
 
         return in_group_call
 
-    async def get_my_volume(self):
-        if not self.full_chat or not self.full_chat.call or not self.my_peer or not self.my_ssrc:
+    async def get_group_participants(
+        self, participants: list = [], sources: list = [], offset: str = "", limit: int = 1
+    ):
+        if not self.full_chat or not self.full_chat.call:
             return
-
         response = await self.client.send(
             functions.phone.GetGroupParticipants(
                 call=self.full_chat.call,
-                ids=[self.my_peer],
-                sources=[int_ssrc(self.my_ssrc)],
-                offset="",
-                limit=1,
+                ids=participants,
+                sources=sources,
+                offset=offset,
+                limit=limit,
             )
         )
-        if response.participants:
-            return response.participants[0].volume
+        return response.participants
```

Comment:
We have participant wrapper. To make the same instance of participant for both mtproto client. Don’t return mtproto client specific objects. You should wrap it using wrapper class. Take a look at handling of updates. Participant update for example